src/react/__tests__/base/scopes.test.tsx

```diff
@@ -73,8 +73,8 @@ it('works', async () => {
     to: fetchUser,
   })
 
-  const user = createStore('guest')
-  const friends = createStore<string[]>([])
+  const user = createStore('guest', {sid: 'user'})
+  const friends = createStore<string[]>([], {sid: 'friends'})
```

Comment:
See comment for https://github.com/effector/effector/pull/655/commits/a2e04ee89b1c3117b3192dc88fff1b189000b85e